lib/twitter/index.js

```diff
@@ -35,10 +36,16 @@ const getTweetById = async id => {
   return {
     id: response.id_str,
     createdAt: response.created_at,
-    text: response.text,
+    html: Autolinker.link(response.text, {mention: 'twitter'}),
     userId: response.user.id_str,
     userName: response.user.name,
-    userScreenName: response.user.screen_name
+    userScreenName: response.user.screen_name,
+    userProfileImageUrl: response.user.profile_image_url_https,
+    // TODO: Pipe multiple images through.
```

Comment:
IMO we will append any `response.entities` except the first image to `html` as links